tests/timeline_tests/test_dashboard_sql_functions.py

```diff
@@ -354,7 +404,10 @@ def get_test_case_data(start, end):
         get_data([fij]),
         '["C","G"]',
     )
-
+def get_test_case_data2(start, end):
```

Comment:
I'm not sure what the purpose of this function is - I think the place it is used, it could just be replaced with the tuple `(start, end)`.

Reply:
Unnecessary method removed.